validate.py

```diff
@@ -1,21 +1,32 @@
+#!/usr/bin/env python
+
+import argparse
+
 from dcctools.config import Configuration
 
 
-def validate():
-    config = Configuration("config.json")
-    return do_validate(config)
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--config",
+        default="config.json",
+        help='Configuration file (default "config.json")',
+    )
+    args = parser.parse_args()
+    return args
 
 
 def do_validate(c):
-    missing_files = c.validate_all_present()
+    missing_files = set(c.validate_all_present())
```

Comment:
Do you think it's possible to make this count unexpected files as well, without a major refactor? I think it might be nice to put a message along the lines of "WARNING: Individual and household hashes appear to be present at the same time" but not if it requires refactoring everything

Reply:
Yes, I'd thought of that and agree that it's a good idea. Shouldn't be too hard to add.

Reply:
Added checks in commit immediately below; focused on `inbox_folder` alone, though. Let me know if you want this check to be done through the rest of the folders.